pkgs/tools/system/uefitool/old-engine.nix

```
@@ -14,19 +9,14 @@
 }:
 
 mkDerivation rec {
-  passthru = {
-    inherit version;
-    inherit sha256;
-    inherit installFiles;
-  };
   pname = "uefitool";
-  inherit version;
+  version = "0.28.0";
 
   src = fetchFromGitHub {
-    inherit sha256;
+    sha256 = "1n2hd2dysi5bv2iyq40phh1jxc48gdwzs414vfbxvcharcwapnja";
```

```suggestion
    hash = "...";
```
this will still be a sha hash, but FOD hashes are supposed to be defined via `hash` now.